src/services/auth.ts

```
@@ -596,8 +849,11 @@ class CasdoorService {
                 console.warn('Error removing cookies:', error);
             }
     
-            // 清除 PKCE 状态
+            // Clear PKCE state
             sessionStorage.clear();
+            
+            // Signal that logout is complete
+            return Promise.resolve();
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)